### PR TITLE
use forecast.io to get forcast

### PR DESCRIPTION
**Pull request by Alexander-Wen**
Do not merge this. This is just to see if everyone is satisfied with the way I am setting up the backend

### CODE REVIEW

File: app/routes/morning/morning.controller.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  const parameters = this.request.body;
+
+  // this will be replaced with the request from the frontend
+  const testParameters = {
+    'lat': 33,
+    'long': 100,
+    'units': celcius
+  };
+
+  this.body = yield weather.getWeather(testParameters);
```
Comment (Alexander-Wen) [inline app/routes/morning/morning.controller.js:15]: the response will be cleaned based on the data the user wants

File: app/services/weather.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const bluebird = require('bluebird');
+const Forecast = require('forecast');
```
Comment (Alexander-Wen) [inline app/services/weather.js:4]: https://www.npmjs.com/package/forecast if anyone wants to check it out

File: package.json [hunk 3]
```diff
@@ -19,6 +19,8 @@
   },
   "homepage": "https://github.com/ProjectReflect/reflect-api#readme",
   "dependencies": {
+    "bluebird": "^3.2.2",
```
Comment (Alexander-Wen) [inline package.json:22]: for promises

File: package.json [hunk 4]
```diff
@@ -19,6 +19,8 @@
...
+    "forecast": "^0.2.1",
```
Comment (Alexander-Wen) [inline package.json:23]: for weather

**Comment by AdrianMachado** [comment]
Shipit

**Comment by timmui** [comment]
:shipit:

**Pull request merged**